### ISSUE TITLE
How can I add arguments to ComplexObject fields?

### ISSUE THREAD

**Issue by onx2** [async-graphql/async-graphql#505]
I'd like to nest a query and pass in arguments to it. I'm using `ComplexObject` to add custom-resolved fields. I'd like the api consumer to be able to write something like the query below. I'm also curious if this is a common pattern or if it makes more sense to just use a `locations` field and have the consumer filter it?
```graphql
query Organization {
  organization(id: "some-uuid") {
    id
    location(id: "some-other-uuid") {
      id
      name
    }
  }
}

```

```rust
#[derive(InputObject)]
pub struct LocationInput {
    pub id: Uuid,
}

#[ComplexObject]
impl Organization {
    pub async fn locations(&self, ctx: &Context<'_>) -> Result<Vec<Location>> {
        Ok(LocationsModel::find_all_by_organization_id(ctx, self.id).await?)
    }

    pub async fn location(
        &self,
        ctx: &Context<'_>,
        input: LocationInput,
    ) -> Result<Option<Location>> {
        let LocationInput { id } = input;
        Ok(LocationsModel::find_by_id(ctx, id).await?)
    }
}
```

I'm receiving this error:
```bash
missing field `is_secret` in initializer of `async_graphql::registry::MetaInputValue`

missing `is_secret`rustc(E0063)
mod.rs(14, 1): missing `is_secret`
```

**Comment by sunli829** [comment]
This problem has been fixed in `v2.8.4`.

**Comment by onx2** [comment]
Perfect, thank you!
